lib/users/routes.js

```diff
@@ -30,18 +35,203 @@ const all = (req, res, next) => {
 
 /**
  * Add a new User
+ * @param {object} req - HTTP Request
+ * @param {object} res - HTTP Response
+ * @param {object} next - Express callback
+ *
+ * @returns {promise} content-types promise
  */
-const add = () => {};
+const add = (req, res, next) => {
+  const errors = _.get(req.session, 'form.users.save.errors', {});
+  const values = _.get(req.session, 'form.users.save.content', {});
+  const merged = req.app.get('users-model') || {};
```

Comment:
On user add pages, password is required, but in the actual implementation it is not